packages/visage/src/components/Input.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {
+  ReactElement,
+  ReactNode,
+  ComponentPropsWithRef,
+  useMemo,
+} from 'react';
+import { StyleSheet } from '@byteclaw/visage-core';
+import { createComponent } from '../core';
+import { Box } from './Box';
+import { StyleProps } from '../createNPointTheme';
```

Comment:
musime to typovat takto z konkretnej temy?

Reply:
Ano pretože lebo to je použité takto aj v ostatnych komponentách... ak užívatelia chcú iné stylery atď musia vytvoriť si na to vlastné nastavenia.